spec/requests/duplicate_logs_controller_spec.rb

```diff
@@ -2,111 +2,296 @@
 
 RSpec.describe DuplicateLogsController, type: :request do
   let(:page) { Capybara::Node::Simple.new(response.body) }
-  let(:user) { create(:user) }
-
-  context "when a user is signed in" do
-    let(:lettings_log) do
-      create(
-        :lettings_log,
-        :completed,
-        created_by: user,
-      )
+  let(:user) { create(:user, :data_coordinator) }
+  let(:lettings_log) do
+    create(
+      :lettings_log,
+      :completed,
+      created_by: user,
+      owning_organisation: user.organisation,
```

Comment:
```suggestion
      created_by: user,
```
as above, this is sufficient